charts/karmada-operator/README.md

```diff
@@ -0,0 +1,71 @@
+# karmada-operator
+
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: latest](https://img.shields.io/badge/AppVersion-latest-informational?style=flat-square)
+
+The Karmada operator is a method for installing, upgrading, and deleting Karmada instances. It builds upon the basic Karmada resource and controller concepts, provides convenience to centrally manage entire lifecyle of Karmada instances in a global cluster. With the operator, you can extend Karmada with custom resources (CRs) to manage your instances not only in local
+clusters but also in remote clusters.
+
+## TL;DR
+
+Switch to the `root` directory of the repo.
+
+```console
+helm install karmada-operator -n karmada-system --create-namespace --dependency-update ./charts/karmada-operator
```

Comment:
This command on line 13 has 90% similarity with that on line 31. Could we add more description?

Reply:
👌